sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/EventPosition.cs

```diff
@@ -16,26 +16,26 @@ namespace Azure.Messaging.EventHubs.Consumer
     public struct EventPosition : IEquatable<EventPosition>
     {
         /// <summary>The token that represents the beginning event in the stream of a partition.</summary>
-        private const string StartOfStreamOffset = "-1";
+        private const string StartOfStream = "-1";
 
         /// <summary>The token that represents the last event in the stream of a partition.</summary>
-        private const string EndOfStreamOffset = "@latest";
+        private const string EndOfStream = "@latest";
```

Comment:
We can't use sequence number as the start of the latest event in the stream yet - Event Hubs hasn't released this yet

Reply:
I don't think we need to change these regardless.   My understanding is that the service team is leaving the special offset values as legal even when replication is enabled.

Reply:
Yeah, the values don't need to change, I probably should've commented this on L30 and L38 - those eventually should be changed to FromSequenceNumber(StartOfStream)/etc. I believe